### ISSUE TITLE
Can you send a message to a recommended user though it's not a match?

### ISSUE THREAD

**Issue by eayin2** [charliewolf/pynder#95]

**Comment by h-2-0** [comment]
I don't think so.  Judging by the URL tree structure an user can only send messages to his/her own matches (i.e. post("/user/matches/{}".format(user)) . In addition, the new 'share' button feature lets a user send a link to another Tinder user and is up to this third-party user to right or left swipe on him/her.
